README.md

```diff
@@ -30,31 +30,39 @@ docker compose up -d
 Install the [Firebase tools](https://firebase.google.com/docs/emulator-suite):
 
 ```sh
-# Install the Firebase CLI using npm, since the curl script is broken (on 06/09/2024)
-npm install -g firebase-tools
+# Install the Firebase CLI
+curl -sL https://firebase.tools | bash
```

Comment:
I believe if default installations work, we should maybe use the OS packages managers to install Firebase instead of piping to she'll. We won't be covering all systems, but I think at least indicating the brew command for macOS and linking the the docs for other operating system would be better.

Reply:
As discussed with @apognu and @Pascal-Delange, we can focus our main documentation on our customers ([Marble repository](https://github.com/checkmarble/marble)), while keeping the submodule documentation (backend and frontend) intended for the internal development team.

Let’s add a disclaimer to this README file to clarify that we officially support the developer experience on macOS only.

Reply:
done

Reply:
I am still of the opinion we should advertise `brew install firebase-cli` for this instead of piping to shell, which is considered a dubious way to install software at best or a security issue at worst.

Reply:
I went with the automatic install script since the homebrew formula is not proposed in [the official doc](https://firebase.google.com/docs/cli#install-cli-mac-linux).
Also our old doc said "curl script".
What do you think if I add the homebrew formula as an alternative ?

Reply:
I'd say keep the official way, and let's not muddy the waters by having too many options. 👍